src/components/relationships-panel/RelationshipsPanel.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <v-card flat id="relationship-style">
+    <div class="form__row three-column mt-3">
```

Comment:
Please verify that your component looks correct in Storybook. For example, I believe "three-column" is not declared as a base style in this repo -- you can add it.

Reply:
Component looks correct in Storybook. I will move three-column to the base style.

Reply:
See my comment below (gray-card).